### PR TITLE
Added Table of Contents

### PR DESCRIPTION
**Pull request by LoopFruits**
Easy for users to navigate to different sections.

Documentation only PR, no builds affected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+==================
+
+  * [Getting Started](#getting-started)
+  * [Project Setup](#project-setup)
+  * [Customize configuration](#customize-configuration)
+  * [App Walkthrough](#app-walkthrough)
+  
+  
+  
+
```
Comment (paramsiddharth) [inline README.md:13]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # amplify
 
+Table of Contents
+==================
```
Comment (paramsiddharth) [inline README.md:4]:
```suggestion
## Table of Contents
```

File: README.md (hunk 1 above)
Comment (paramsiddharth) [inline README.md:13]:
```suggestion
```

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
There are some minor changes needed for consistency. The rest looks good!

**Review by paramsiddharth — APPROVED** [review]

**Comment by JimmyJammer** [comment]
FWIW, GitHub automatically generates a Table of Contents for these if you just click on the upper-left menu button near the filename in the viewing panel:

Screen Shot 2022-10-20 at 10 47 41 AM

The beauty of GitHub's automatic solution is that it will never become out-of-date. ✨ 

With that in mind, do we still want to add this? It is undoubtedly more visible, so it's just a question of if that visibility outweighs the long-term maintenance burden. 🤷‍♂️ 🤔

**Comment by paramsiddharth** [comment]
> @JimmyJammer FWIW, GitHub automatically generates a Table of Contents for these if you just click on the upper-left menu button near the filename in the viewing panel:
> 
> Screen Shot 2022-10-20 at 10 47 41 AM
> 
> The beauty of GitHub's automatic solution is that it will never become out-of-date. ✨
> 
> With that in mind, do we still want to add this? It is undoubtedly more visible, so it's just a question of if that visibility outweighs the long-term maintenance burden. 🤷‍♂️ 🤔

Good point! As amazing as this feature is, I think we'd still go with considering this PR for this particular repository. "More visibility" isn't exactly going to be the criterion, but I think the design team intended on having this added to the README.

Nevertheless, thank you for pointing out the fact that there's a button that automatically generates the ToC for us! It will now be helpful for me every time I navigate through GitHub repositories. 🤗

**Comment by JamesMGreene** [comment]
Forgot I was logged into my test account there. 😊

**Comment by paramsiddharth** [comment]
> Forgot I was logged into my test account there. 😊

@JamesMGreene Haha, big reveal, James! 😁 Your test account just got a new follower~! Had I known it was you, I'll probably let you make the final decision instead of what was proposed in the bugbash. Sorry about that!

**Comment by JamesMGreene** [comment]
I'm fine with the change! 👍🏻

**Pull request merged**